.github/workflows/ci.yml

```
@@ -2,25 +2,23 @@ name: Android CI
-
+    branches: ["ci/cd", "master"]
 jobs:
   build:
     runs-on: ubuntu-latest
 
 
     steps:
       - uses: actions/checkout@v3
-      - name: set up JDK 11
+      - name: set up JDK 17
         uses: actions/setup-java@v3
```

actionlint (1.7.8)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)